### PR TITLE
First pass at Service Maps in the Opsramp API

### PR DESCRIPTION
**Pull request by lenihan01**

### CODE REVIEW

File: opsramp/service_maps.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+from opsramp.base import ApiWrapper
+
+
+class Service_Maps(ApiWrapper):
```
Comment (jofegan) [inline opsramp/service_maps.py]: conventionally would not have _ in a python class name. more typical like ServiceMaps

File: opsramp/service_maps.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+#
+# A minimal Python language binding for the OpsRamp REST API.
+#
+# sites.py
+# Classes dealing directly with OpsRamp Sites.
+# Sites are used to organise resources by geographical location.
```
[the remark this answers is not in the document]
Comment (lenihan01) [reply opsramp/service_maps.py]: Should read better now.

File: opsramp/service_maps.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+from __future__ import print_function
+from opsramp.base import ApiWrapper
+
+
+class Service_Maps(ApiWrapper):
+    def __init__(self, parent):
+        super(Service_Maps, self).__init__(parent.api, 'serviceGroups')
+
+    def get(self, uuid=False, minimal=False):
```
[the remark this answers is not in the document]
Comment (lenihan01) [reply opsramp/service_maps.py]: I think using 'None' broke the logic when I tried it so I would like to leave it as-is unless
it is very important to change.

File: samples/service_maps_get.py [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+
+def connect():
+    url = os.environ['OPSRAMP_URL']
+    key = os.environ['OPSRAMP_KEY']
+    secret = os.environ['OPSRAMP_SECRET']
+    return opsramp.binding.connect(url, key, secret)
+
+
+def walk(map):
```
[the remark this answers is not in the document]
Comment (lenihan01) [reply samples/service_maps_get.py]: I had yaml dump here initially but found the json dumps easier to read. 
Easy to change if needed.

File: opsramp/service_maps.py (hunk 1 above)
Comment (lenihan01) [reply opsramp/service_maps.py]: fixed

**Pull request merged**